README.md

```diff
@@ -231,17 +239,16 @@ implementation, it is by far the most effective way to see an RFC through to
 completion: authors should not expect that other project developers will take on
 responsibility for implementing their accepted feature.
 
-Minor modifications to accepted RFCs can be done in follow-up pull requests. We
-strive to write each RFC in a manner that it will reflect the final design of
-the feature; but the nature of the process means that we cannot expect every
-merged RFC to actually reflect what the end result will be after implementation.
-
-In general, once accepted, RFCs should not be substantially changed. Only very
-minor changes should be submitted as amendments. More substantial changes should
-be new RFCs, with a note added to the original RFC. Exactly what counts as a
-"very minor change" is up to the RFC Shepherd Team of the RFC to be amended, to
-be decided in cooperation with the RFC Steering Committee.
-
+RFC documents are intended to be seen as the documentation of a decision and a
```

Comment:
As a follow-up, we might want to rename the RFC to ADR or DR to remove the confusion even further.

The document is called an RFC because it's looking to gather comments. Once all the comments have been gathered and integrated, what we have is an (Architecture) Record of Decision.

Splitting those two might allow for more flexibility in the process. This repo becomes a log of record of decision, and the RFC process can potentially happen in a different place.

![image](https://github.com/NixOS/rfcs/assets/3248/cccbfa70-c982-44c2-9771-995ae78d1491)

Reply:
I agree that RFC is a pretty bad name (even for its first incarnation IMO), and that projects who pick something "change proposal" or "enhancement proposal" have better names. But what you are suggesting is not a name change but a fundamental new vision of how our process might look like, and therefore requires to have its own RFC discussion.

(As you can see, I'm trying pretty hard on making an incremental improvement to the current situation without getting involved in yet another RFC …)

Reply:
Agreed, this is why I wrote: "as a follow-up". It was more of a general remark.